weave

```diff
@@ -1760,6 +1785,11 @@ EOF
             echo "       Service: proxy"
             echo "       Address: $PROXY_ADDRS"
         fi
+        if [ -z "$SUB_STATUS" ] && check_running $PLUGIN_CONTAINER_NAME 2>/dev/null ; then
+            echo
+            echo "       Service: plugin"
+            echo "       Address: unix:///run/docker/plugins/weave.sock"
```

Comment:
This information isn't very useful to the user, is it?  How about "Driver name: weave" ?